Configuration.cs

```diff
@@ -10,26 +10,27 @@ public class Configuration : ModConfig
 		public override ConfigScope Mode => ConfigScope.ServerSide;
 
 		public static Configuration Instance;
-
+		//Label & Tooltip deprecated. Use Keys or just specify as Args: it will be interpreted as a key if beginning starts with "$".
+		//https://github.com/tModLoader/tModLoader/pull/3302
 		[DefaultValue(true)]
-		[Label("Disable Tombstones")]
-		[Tooltip("Prevents tombstones from being spawned when you die\nDefaults to true")]
+		[LabelArgs("$Disable Tombstones")]
```

Comment:
After further testing the labels and tooltips do not work with Args. A ModConfig class should be made with appropriate keys. Continue using label and tooltip or move to ModConfig class.